x/crosschain/types/cctx.go

```diff
@@ -145,6 +145,7 @@ func (m *CrossChainTx) AddRevertOutbound(gasLimit uint64) error {
 			GasLimit: gasLimit,
 		},
 		TssPubkey: m.GetCurrentOutboundParam().TssPubkey,
+		CoinType:  m.InboundParams.CoinType,
```

Comment:
This change is out of scope for this pr but might be small enough that it can be included here